README.md

```diff
@@ -86,9 +86,9 @@ It can automatically generates the corresponding Dashboard configurations based
 
 ## Sources
 
-* [Guide](https://ava.antv.vision/en/docs/guide/intro)
-* [API](https://ava.antv.vision/en/docs/api)
-* [Examples](https://ava.antv.vision/en/examples/gallery)
+* [Guide](https://ava.antv.antgroup.com/en/docs/guide/intro)
+* [API](https://ava.antv.antgroup.com/en/docs/api)
+* [Examples](https://ava.antv.antgroup.com/en/examples/gallery)
```

Comment:
antgroup.com is primary, antv.vision is backup, does the team agree about the issue?

Reply:
that is fine